### PR TITLE
Bug: delete contribution link

### LINKED ISSUE #2109: 🐛 [Bug] Typo im Dialog "Meine Beiträge"
## 🐛 Bugreport

Im Dialog "Meine Beiträge" wird ein Text ausgegeben, in dem noch zwei Typos enthalten sind:

IST:  Es wird bald an dieser Stelle die Möglichkeit geben das ein Dialog zwischen Moderatoren und dir stattfinden kann. Solltest du jetzt Probleme haben bitte nimm Kontakt mit dem Support auf. 
SOLL: Es wird bald an dieser Stelle die Möglichkeit geben, dass ein Dialog zwischen Moderatoren und dir stattfinden kann. Solltest du jetzt Probleme haben, bitte nimm Kontakt mit dem Support auf. 

Die Stellen im Detail:
IST:     ...geben das ...
SOLL: ...geben, dass...
  und
IST:     ...Probleme haben bitte...
SOLL: ...Probleme haben, bitte...

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
a contribution link can be deleted. 
Automatic reloading of the contribution list when a delete or create has run.

### Issues
- fixes #2109 
 
 

https://user-images.githubusercontent.com/1324583/190616911-e277b3a3-ef9d-4b0d-b7ed-1ef01cbf311c.mp4

### CODE REVIEW

File: admin/src/locales/de.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "contributionLinks": "Beitragslinks",
     "create": "Anlegen",
     "cycle": "Zyklus",
-    "deleteNow": "Automatische Creations wirklich löschen?",
+    "deleteNow": "Automatische Creations '{name}' wirklich löschen?",
```
Comment (Mogge) [inline admin/src/locales/de.json:10]:
```suggestion
    "deleteNow": "Automatische Schöpfung '{name}' löschen?",
```

File: admin/src/locales/en.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "contributionLinks": "Contribution Links",
     "create": "Create",
     "cycle": "Cycle",
-    "deleteNow": "Do you really delete automatic creations?",
+    "deleteNow": "Do you really delete automatic creations '{name}'?",
```
Comment (Mogge) [inline admin/src/locales/en.json:10]:
```suggestion
    "deleteNow": "Delete automatic creations '{name}'?",
```

File: admin/src/components/ContributionLinkList.vue [hunk 3]
```diff
@@ -70,28 +70,30 @@ export default {
...
+          if (value)
+            await this.$apollo
+              .mutate({
+                mutation: deleteContributionLink,
+                variables: {
+                  id: id,
+                },
+              })
+              .then(() => {
+                this.toastSuccess('TODO: request message deleted ')
```
Comment (Mogge) [inline admin/src/components/ContributionLinkList.vue:90]: Please have a message

Comment (Mogge) [reply admin/src/components/ContributionLinkList.vue:90]: Automatische Schöpfung gelöscht!

File: admin/src/components/ContributionLink.vue [hunk 4]
```diff
@@ -15,7 +15,10 @@
       <b-collapse v-model="visible" id="newContribution" class="mt-2">
         <b-card>
           <p class="h2 ml-5">{{ $t('contributionLink.contributionLinks') }}</p>
-          <contribution-link-form :contributionLinkData="contributionLinkData" />
+          <contribution-link-form
+            :contributionLinkData="contributionLinkData"
+            @get-contribution-links="getContributionLinks"
```
Comment (Mogge) [inline admin/src/components/ContributionLink.vue:20]:
```suggestion
            @get-contribution-links="$emit('get-contribution-links')"
```

File: admin/src/components/ContributionLink.vue [hunk 5]
```diff
@@ -24,6 +27,7 @@
           v-if="count > 0"
           :items="items"
           @editContributionLinkData="editContributionLinkData"
+          @get-contribution-links="getContributionLinks"
```
Comment (Mogge) [inline admin/src/components/ContributionLink.vue:30]:
```suggestion
          @get-contribution-links="$emit('get-contribution-links')"
```

File: admin/src/components/ContributionLink.vue [hunk 6]
```diff
@@ -61,6 +65,9 @@ export default {
       if (!this.visible) this.$root.$emit('bv::toggle::collapse', 'newContribution')
       this.contributionLinkData = data
     },
+    getContributionLinks() {
+      this.$emit('get-contribution-links')
+    },
```
Comment (Mogge) [inline admin/src/components/ContributionLink.vue:70]:
```suggestion
```

**Review by Mogge — CHANGES_REQUESTED** [review]

File: admin/src/components/ContributionLinkList.vue (hunk 3 above)
Comment (ogerly) [reply admin/src/components/ContributionLinkList.vue:90]: done

**Review by Mogge — APPROVED** [review]

**Pull request merged**